src/utils/recordsCardUtils.js

```diff
@@ -1,28 +1,83 @@
 const recordsCardUtils = {
     methods: {
-        getButtonLabel(ItemTitle) {
-            let returnedTitle;
-            returnedTitle = ItemTitle;
-            switch (ItemTitle) {
-                case 'taxonomies':
-                    returnedTitle = 'species';
-                    break;
-                case 'userDefinedTags':
-                    returnedTitle = 'tags';
-                    break;
-                default:
-                    returnedTitle = ItemTitle;
-                    break;
-            }
-            return returnedTitle;
-        },
         getRecordLink(record) {
             if (record.doi) {
                 // Only the last part of the DOI is needed for local links.
                 return record.doi.split('/').pop();
             } else {
                 return record.id;
             }
+        },
+        getChipColor(chip){
+            if(chip.type==='subjects'){
+                return 'subject_color';
+            }
+            else if(chip.type==='domains') {
+                return 'domain_color';
+            }
+            else if(chip.type==='taxonomies') {
+                return 'taxonomic_color';
+            }
```

Comment:
You need to harmonize these with the editor colors.
You can find them in store/editor.js -> state.colors

Reply:
tnx.recommended is removed based on the latest approved mocks(the decision was made) but there are chances yet in the future to use them but not in a ribbon way. about the editor color I check that.